samples/react-m365-services-health/README.md

```diff
@@ -0,0 +1,98 @@
+# Microsoft 365 Services Health
```

Comment:
Try renaming it "Service Health for Microsoft 365" to comply with Microsoft branding guidelines.